Add orchestrator explorer as a prereq for SCA (#19595)
DataDog/documentation: content/en/cloud_cost_management/container_cost_allocation.md

1. Set up and configure [AWS Cloud Cost integration][1].
2. (Optional) Enable [AWS Split Cost Allocation][5] for usage-based ECS allocation.
1. At least one of the following must be running:

>     - [**Datadog Agent**][2] in a Kubernetes environment using EC2 instances. Ensure that the [**Orchestrator Explorer**][6] is enabled in your Agent configuration.

Reviewer: ```suggestion
    - [**Datadog Agent**][2] in a Kubernetes environment using EC2 instances. Ensure that you enable the [**Orchestrator Explorer**][6] in your Agent configuration.
```

>     - [**Datadog Agent**][2] in a Kubernetes environment using EC2 instances. Please also ensure that the [**Orchestrator Explorer**][6] is enabled in your agent configuration.

Reviewer: ```suggestion
    - [**Datadog Agent**][2] in a Kubernetes environment using EC2 instances. Please also ensure that the [**Orchestrator Explorer**][6] is enabled in your Agent configuration.
```

>     - [**Datadog Agent**][2] in a Kubernetes environment using EC2 instances. Please also ensure that the [**Orchestrator Explorer**][6] is enabled in your Agent configuration.

Reviewer: ```suggestion
    - [**Datadog Agent**][2] in a Kubernetes environment using EC2 instances. Ensure that the [**Orchestrator Explorer**][6] is enabled in your Agent configuration.
```

[3]: /containers/amazon_ecs/?tab=awscli
[4]: https://docs.aws.amazon.com/cur/latest/userguide/what-is-cur.html
[5]: https://docs.aws.amazon.com/cur/latest/userguide/enabling-split-cost-allocation-data.html

> [6]: https://docs.datadoghq.com/infrastructure/containers/orchestrator_explorer/?tab=datadogoperator

Reviewer 2: i think this can be a relative link (see [3] above)
